Model/EventsForThirdPartyModules.php

```diff
@@ -42,6 +43,14 @@ class EventsForThirdPartyModules
                     "boltClass" => Aheadworks_Giftcard::class,
                 ],
             ],
+        ],
+        'applyIdMeToCustomerSession' => [
```

Comment:
Here, some more generic name, similar to "afterLoadSession" and "beforeDeleteOrder", instead of "applyIdMeToCustomerSession"

Reply:
I've updated the code